dev/snippets/config/templates/stateful_widget_animation.tmpl

```diff
@@ -31,7 +30,7 @@ class MyStatefulWidget extends StatefulWidget {
 }
 
 // This is the private State class that goes with MyStatefulWidget.
-class _MyStatefulWidgetState extends State<MyStatefulWidget> {
+class _MyStatefulWidgetState extends State<MyStatefulWidget> with TickerProviderStateMixin {
```

Comment:
we definitely don't want every example to do that. Even most examples that want to do a ticker would only use SingleTickerProviderStateMixin...

Reply:
This isn't _every_ example - just those that use the `stateful_widget_animation` template (which right now is only one).